src/components/SavedTemplates.tsx

```diff
@@ -1,19 +1,25 @@
 import React, { useState } from 'react'
+import { CurrentTemplate } from '../pages/Payslip';
 import S from 'styled-components';
 
-const SavedTemplates = ({useTemplate}) => {
+interface SavedTemplatesProps {
+  useTemplate: (template: CurrentTemplate) => void;
+}
+
+const SavedTemplates:React.FC<SavedTemplatesProps> = ({useTemplate}) => {
   const[clearallTemplatedMsg, setClearedAllTemplatesMsg] = useState('');
   const[templatedIsUsedMsg, setTemplateIsUsedMsg] = useState('');
 
-  const templates = JSON.parse(localStorage.getItem('templates')) ?? [];
-
-  const handleUseTemplate = (template) => {
+  const pullTemplates = localStorage.getItem('templates');
+  const templates = pullTemplates !== null ? JSON.parse(pullTemplates) : [];
+ console.log(templates);
```

Comment:
remove

Reply:
@itmilos   I have made the recommended changes.
Please test the branch before merging into master. It is a  big change and also configuration changes